packages/common/src/search/_internal/hubSearchItemsHelpers/ogcItemToDiscussionPostResult.ts

```diff
@@ -30,5 +30,41 @@ export async function ogcItemToDiscussionPostResult(
     rawResult: ogcItem.properties,
     access: null,
     family: null,
+
+    // Post properties
+    channelId: ogcItem.properties.channelId,
```

Comment:
I don't know that we want to expose all these top level properties on the interface, especially since they only belong to a single entity type. Is there a problem with having UI layer just use `rawResult.<propName>` instead? I'm curious if @dbouwman feels the same.

Reply:
Since `IHubSearchResult` has `[key: string]: any;` it will allow any props to be added. By I'd agree with @sonofflynn89 that we'd be better off copying `ogcItem.properties` to the returned object as `.rawResult`. So we've have `result.rawResult.postType` === `ogcItem.properties.postType`

Reply:
@drspacemanphd in hindsight, there may have been a better pattern to utilize, but the hubSearch + gallery have evolved over time, and this is where we find ourselves. This is the short version of the history...

The initial design was to have `hubSearch` return objects optimized for display in the client, and abstracted from their implementation - thus `IHubSearchResult`. Then there was a request to allow the system to return arbitrary additional metadata - you've heard this from Andrew as "percent cloud cover" etc - so we added `[key: string]: any;` to the search result to allow for that. Later on, we got to a place where different galleries needed to exert more specific control over what's displayed in the cards in the gallery (implemented as a callback fn that can be passed to the gallery), and to do this, we needed to expose more of the underlying source object - thus `.rawResult`.